src/orders/components/OrderPaymentOrTransaction/OrderPaymentOrTransaction.test.tsx

```diff
@@ -27,6 +27,23 @@ jest.mock("@saleor/macaw-ui", () => ({
   ResponsiveTable: jest.fn(() => <></>),
 }));
 
+jest.mock("@saleor/macaw-ui/next", () => ({
+  useTheme: jest.fn(() => () => ({})),
```

Comment:
**suggestion**: what about wrapping component in `ThemeProvider` instead of mocking?